src/components/touchables/TouchableHighlight.tsx

```diff
@@ -35,7 +38,7 @@ export default class TouchableHighlight extends Component<
     underlayColor: 'black',
   };
 
-  constructor(props: TouchableHighlightProps & GenericTouchableProps) {
+  constructor(props: RNTouchableHighlightProps & GenericTouchableProps) {
```

Comment:
```suggestion
  constructor(props: TouchableHighlightProps) {
```

Is there a reason to leave an explicit union here instead of using the new type?

Reply:
Not at all, forgot to update. Thanks for pointing it out!